dashboard/test/components/PackageLocationCard.test.ts

```diff
@@ -35,10 +36,57 @@ describe("PackageLocationCard.vue", () => {
         </div>
       </div>"
     `);
+
+    unmount();
+  });
+
+  it("renders when the package location is moved", async () => {
+    const { getByText, unmount } = render(PackageLocationCard, {
+      global: {
+        plugins: [
+          createTestingPinia({
+            createSpy: vi.fn,
+            stubActions: false,
+            initialState: {
+              package: {
+                current: {
+                  status: api.PackageShowResponseBodyStatusEnum.Done,
+                  location: "perma-aips-1",
+                } as api.PackageShowResponseBody,
+              },
+            },
+          }),
+        ],
+      },
+    });
+
+    getByText("perma-aips-1");
+
+    const packageStore = usePackageStore();
+    packageStore.move.mockImplementation(async () => {
+      packageStore.$patch((state) => {
+        state.current.status =
+          api.EnduroStoredPackageResponseBodyStatusEnum.InProgress;
+        state.locationChanging = true;
+      });
+    });
+
+    vi.mock("../../src/dialogs", () => {
+      return {
+        openPackageLocationDialog: () => "perma-aips-2",
+      };
+    });
+
+    const button = getByText("Choose storage location");
+    await fireEvent.click(button);
```

Comment:
Using await here ensures that we wait for the next tick, see https://github.com/testing-library/vue-testing-library/blob/4a663b9a265c4e2a51e0b550c8d1809169bd1daf/src/fire-event.js#L4-L12.